mitmproxy/tools/web/webaddons.py

```diff
@@ -21,6 +22,10 @@ def load(self, loader):
             "web_host", str, "127.0.0.1",
             "Web UI host."
         )
+        loader.add_option(
+            "columns", Sequence[str], ["", "Path", "Method", "Status", "Size", "Time"],
```

Comment:
Can we call this web_columns (or web_flowlist_columns?) please to make clear that this is about mitmweb only for now? Also let's make the help text a bit more descriptive. How about "Columns to show in the flow list."?